src/mgmt/osp-sah.ks

```diff
@@ -120,48 +120,60 @@ extern_bond_name="CHANGEME e.g. bond1"
 extern_boot_opts="onboot none"
 extern_bond_opts="mode=802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1"
 extern_ifaces="CHANGEME e.g. em2 p1p2"
+extern_bond_mtu="CHANGEME e.g. 9216"
```

Comment:
We shouldn't give a value higher than 9000 since I thought that was the max we were going to support

Reply:
On switch ports, MTU size is set to 9216. Also, all bonds on different nodes carry different VLANs. And bonds MTU should be equal or greater than the VLANs MTU. With the option to give a different MTU to each network,  we set it to MAX possible MTU on the switches.

Reply:
Sambhu has confirmed from RedHat that 9000 is the standard, supported jumbo frame size. The overhead of different encapsulations is handled by the switch's mtu of 9216.